### ISSUE TITLE
nuget 3.x is unable to create native packages due to absolute paths in the .zip

### ISSUE THREAD

**Issue by Malvineous** [NuGet/Home#2832]
If you use the CoApp tools to create a native package, then you have to use version 2.x of `nuget.exe`.  If you use version 3.x then the `.nupkg` file has absolute paths:

```
foo.nuspec
/build/native/foo.targets
/build/native/lib/Win32/Debug/foo.lib
```

This causes Visual Studio to fail when trying to add the package to a project:

```
Could not install package 'foo 1.0'. You are trying to install this package into a project
    that targets 'native,Version=v0.0', but the package does not contain any assembly
    references or content files that are compatible with that framework. For more
    information, contact the package author.
```

I tried unzipping the `.nupkg` file and then rezipping it, with no other changes, and Visual Studio accepts it once the absolute paths are removed:

```
foo.nuspec
build/native/foo.targets
build/native/lib/Win32/Debug/foo.lib
```

This looks like a bug to me, because you would expect either all the paths to be absolute (and VS to accept them) or none of them be absolute, as with NuGet 2.x.

**Comment by rrelyea** [comment]
I believe we've fixed this issue in 3.4.4 -- but that build isn't available yet. The nightly build (3.5.0-rc) of command line that is downloadable from: http://dist.nuget.org/index.html should have the same fix.
Can you please let us know how that nuget.exe works?

**Comment by Malvineous** [comment]
Thanks for the followup!  It looks like this has indeed fixed the problem.

I wasn't sure which version to use (couldn't find 3.5.0-rc) so I tried both 3.4.0-rc (reported as 3.4.4.874) and 3.5.0-beta (reported as 3.5.0.1284), neither of which worked.  But when I tried the NuGet.CommandLine nightly build (reported as 3.5.0.1365), that created the .nupkg file correctly!  No more leading slashes on the `build` paths.

Thanks for fixing this issue!

**Comment by rrelyea** [comment]
3.4.4 NuGet.exe bits to try are here: https://dist.nuget.org/win-x86-commandline/v3.4.4/NuGet.exe 
Can you please try?

**Comment by rrelyea** [comment]
We believe this has been addressed in the now released 3.4.4. If we are mistaken, please reactivate.

**Comment by Malvineous** [comment]
Many thanks - just confirming I have tested with 3.4.4 and the problem is indeed resolved.
